src/test/java/com/thebois/models/beings/ColonyTests.java

```diff
@@ -53,4 +65,179 @@ public void updateCallsUpdateOnAllBeings() {
+        final Colony colony = mockColony();
+
+        // Act
+        colony.add(new Log());
+        colony.add(new Log());
+
+        final int count = colony.numberOf(ItemType.LOG);
+
+        // Assert
+        assertThat(count).isEqualTo(2);
+    }
+
+    @Test
+    public void countIsCorrectValueWhenInventoryDoesNotHaveSpecifiedItemType() {
+        // Arrange
+        final Colony colony = mockColony();
+
+        // Act
+        colony.add(new Log());
+        colony.add(new Log());
+
+        final int count = colony.numberOf(ItemType.ROCK);
+
+        // Assert
+        assertThat(count).isEqualTo(0);
+    }
+
+    private Colony mockColony() {
+        final Collection<IBeing> pawns = new ArrayList<>(0);
+        return new Colony(pawns);
+    }
+
+    @Test
+    public void takeMultipleItemsFromColony() {
+        // Arrange
+        final Colony colony = mockColony();
+
+        // Act
+        colony.add(new Log());
+        colony.add(new Log());
+
+        final ArrayList<IItem> result = colony.takeAmount(ItemType.LOG, 2);
+
+        // Assert
+        assertThat(result.size()).isEqualTo(2);
+        assertThat(result.get(0).getType()).isEqualTo(ItemType.LOG);
+        assertThat(result.get(1).getType()).isEqualTo(ItemType.LOG);
+    }
+
+    @Test
+    public void canNotTakeMultipleItemsFromColony() {
+        // Arrange
+        final Colony colony = mockColony();
+
+        // Act
+        colony.add(new Log());
+        colony.add(new Log());
+
+        final Exception exception = assertThrows(IllegalArgumentException.class, () -> {
+            colony.takeAmount(ItemType.ROCK, 2);
+        });
+
+        // Assert
+        assertThat(exception.getMessage()).isEqualTo(
+            "Not enough of the specified ItemType in the inventory");
+    }
+
+    @Test
+    public void colonyHasItem() {
+        // Arrange
+        final Colony colony = mockColony();
+
+        // Act
+        colony.add(new Log());
+        final boolean result = colony.hasItem(ItemType.LOG);
+
+        assertThat(result).isTrue();
+    }
+
+    @Test
+    public void colonyDoesNotHaveItem() {
+        // Arrange
+        final Colony colony = mockColony();
+
+        // Act
+        final boolean result = colony.hasItem(ItemType.LOG);
+
+        assertThat(result).isFalse();
+    }
+
+    @Test
+    public void colonyHasMultipleOfItem() {
+        // Arrange
+        final Colony colony = mockColony();
+
+        // Act
+        colony.add(new Log());
+        colony.add(new Log());
+
+        final boolean result = colony.hasItem(ItemType.LOG, 2);
+
+        // Assert
+        assertThat(result).isTrue();
+    }
+
+    @Test
+    public void colonyDoesNotHaveMultipleOfItem() {
+        // Arrange
+        final Colony colony = mockColony();
+
+        // Act
+        final boolean result = colony.hasItem(ItemType.LOG, 2);
+
+        // Assert
+        assertThat(result).isFalse();
+    }
+
+    @Test
+    public void addMultipleItemsToColony() {
```

Comment:
For future reference, this way of naming tests is correct - but only halfway to being truly helpful.

Generally it's helpful if the names follow the format: 
`whenSomethingIsDoneAndTheObjectIsInThisStateThenThisHappens`
On a lot of these tests, you've only got the:
`whenSomethingIsDone`, you never describe what the criterias are, or what is expected to happen. 

Now, this is easy to see if I read the code of the test, but from the test runner all you can see is the name and whether it failed or not, so it's much more helpful to read that "`addMultipleItemsWhenInventoryEmptySucceeds` has failed" rather than "`addMultipleItemsToColony` has failed", if that makes sense?

(sidenote, we're in `ColonyTests` so there's nothing but the colony to add items to, no need to have "ToColony" in name)